Various suggestions for the docs (#7423)
netdata/netdata: packaging/installer/README.md

We provide our own flavour of binary packages for the most common operating systems that comply with .RPM and .DEB packaging formats.

Early adopters may experiment with our .DEB formatted packages using our nightly releases.

Our current packaging infrastructure provider is [Package Cloud](https://packagecloud.io).

> **Please Note** that you must add the proper Package Cloud repositories to the system Package sources **before** attempting to install Netdata (e.g `apt-get install netdata`).

Reviewer: ```suggestion
You must add the proper Package Cloud repositories to your system's package sources **before** attempting to install Netdata (e.g `apt-get install netdata`).
```

We provide two separate repositories, one for our stable releases and one for our nightly releases.

1.  Stable releases: Our stable production releases are hosted in [netdata/netdata](https://packagecloud.io/netdata/netdata) repository of package cloud
2.  Nightly releases: Our latest releases are hosted in [netdata/netdata-edge](https://packagecloud.io/netdata/netdata-edge) repository of package cloud

Netdata is **committed** to support installation of our solution to **all operating systems**. This is a constant battle for Netdata, as we strive to automate and make things easier for our users.

> For the operating system support matrix, please visit our [distributions](../../packaging/DISTRIBUTIONS.md) support page.

Reviewer: ```suggestion
For the operating system support matrix, please visit our [distributions support page](../../packaging/DISTRIBUTIONS.md).
```
